src/openms/source/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.cpp

```diff
@@ -683,20 +694,24 @@ namespace OpenMS
       PeptideHit hit = getAnnotation_(c.getPeptideIdentifications());
       for (auto const & f : c.getFeatures())
       {
-        // indices in experimental design are 0-based as the map indices
         //TODO MULTIPLEXED: needs to be adapted for multiplexed experiments
```

Comment:
What about this Todo?

Reply:
Good question. Looks correct to me now. quantifyFeature_ now uses the correct fraction group, fraction, and sample.

```
pep_quant_[seq].abundances[fraction][hit.getCharge()][sample] += feature.getIntensity();
```

We don't consider channel specific effects here and just sum independent of the used channel.